src/backend/joanie/core/models/events.py

```diff
@@ -0,0 +1,47 @@
+"""
+Declare and configure the models for Joanie's events
+"""
+import logging
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from joanie.core import enums
+from joanie.core.models.base import BaseModel
+
+logger = logging.getLogger(__name__)
+
+
+class Event(BaseModel):
+    """
+    Event represents a message to a user
+    """
+
+    user = models.ForeignKey(
+        to="core.User",
+        verbose_name=_("user"),
+        related_name="events",
+        on_delete=models.CASCADE,
+    )
+    message = models.TextField(_("message"))
+    level = models.CharField(
+        _("level"),
+        max_length=20,
+        choices=enums.EVENT_LEVEL_CHOICES,
+        default=enums.EVENT_INFO,
+    )
```

Comment:
I wonder if we should not store only event `type` instead then delegate the responsability to choose the message to display to mail view or frontend.

Reply:
Indeed, the message purpose was not to be displayed in frontend, but it was just informative.
I'll remove it for clarity.